dipy/reconst/csdeconv.py

```diff
@@ -964,7 +964,7 @@ def recursive_response(gtab, data, mask=None, sh_order=8, peak_thr=0.01,
     where_dwi = lazy_index(~gtab.b0s_mask)
     response_p = np.ones(len(n))
 
-    for num_it in range(1, iter):
+    for num_it in range(iter):
```

Comment:
That would break the backward compatibility of the function (it's an input), including the script I am revamping which lead me to see this. Unless we add deprecation warning and everything to it it might break more stuff than it could fix.